quickstart/kubernetes-guestbook.md

```diff
@@ -0,0 +1,373 @@
+---
+title: "Create a Guestbook with Redis and PHP"
+---
+
+In this tutorial, we'll build and deploy
+[the standard Kubernetes Guestbook example](https://github.com/kubernetes/examples/tree/master/guestbook) using Pulumi.
+
+The guestbook is a simple, multi-tier web application that uses Redis and Nginx, powered by Docker containers and
+Kubernetes. The primary difference between this example and the standard Kubernetes one is that we'll be authoring it in
+TypeScript instead of YAML, and we'll deploy it using `pulumi` rather than `kubectl`. This gives us the full power of
+real languages, combined with immutable infrastructure with diffable updates, alongside reliable orchestration, status,
+and robust state management.
```

Comment:
Perhaps we could simplify this last sentence?

Reply:
:+1: